bot.rb

```diff
@@ -0,0 +1,63 @@
+require 'dotenv/load'
+require 'nokogiri'
+require 'telegram/bot'
+require 'open-uri'
+require 'logger'
+
+TOKEN = ENV['TG_TOKEN'].freeze
+URL = 'http://bash.im/best'.freeze
+$posts = []
+
+def initialize
```

Comment:
this method looks strange, when I say that we need to move logger to initializer, I meant that we need to path logger object to bot initializer here: Telegram::Bot::Client.run(TOKEN)